config.json

```diff
@@ -1441,6 +1441,18 @@
       "difficulty": 0,
       "topics": null,
       "deprecated": true
+    },
+    {
+      "slug": "grains",
+      "uuid": "5ee66f39-5e37-4907-a6d9-f55d38324c6c",
+      "core": false,
+      "unlocked_by": "two-fer",
+      "difficulty": 2,
+      "topics": [
+        "loops",
+        "math"
+      ],
+      "deprecated": false
     }
```

Comment:
What I wanted to say was is that in this file, we group exercises by difficulty level.
You put this new exercise at the end of the file, put it should be placed with the exercises of level 2, for example on line 97.

Reply:
Ah, got it. I was also wondering if what was the ordering of this file, as it was not by alphabetical order.